setup.py

```diff
@@ -58,23 +58,13 @@ def get_extras_require():
         'bokeh', 'chainer>=5.0.0', 'cma', 'keras', 'lightgbm', 'mock',
         'mpi4py', 'mxnet', 'pandas', 'plotly>=4.0.0', 'pytest', 'scikit-optimize',
         'tensorflow', 'tensorflow-datasets', 'xgboost', 'scikit-learn>=0.19.0',
-        'torch', 'torchvision', 'pytorch-ignite',
-
-        # TODO(ohta):
-        # Remove this version constraint once
-        # https://github.com/williamFalcon/pytorch-lightning/issues/469 is resolved.
-        'pytorch_lightning<0.5.3',
+        'torch', 'torchvision', 'pytorch-ignite', 'pytorch_lightning',
```

Comment:
A nit that I just noticed, although not introduced by this PR but maybe something I introduced before... (I think this underscore is treated as an invalid character by setuptools and automatically converted to a dash.)
```suggestion
        'torch', 'torchvision', 'pytorch-ignite', 'pytorch-lightning',
```

Reply:
Thanks! Merged.